llvm/lib/ObjCopy/ELF/ELFObjcopy.cpp

```diff
@@ -215,23 +215,41 @@ static Error dumpSectionToFile(StringRef SecName, StringRef Filename,
 }
 
 Error Object::compressOrDecompressSections(const CommonConfig &Config) {
-  // Build a list of the debug sections we are going to replace.
+  // Build a list of sections we are going to replace.
   // We can't call `AddSection` while iterating over sections,
   // because it would mutate the sections array.
```

Comment:
Something looks a little off with this comment now, possibly as a result of previous changes. It might just be that `AddSection` should be `addSection`.

Reply:
Fixed